google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -1394,80 +1376,6 @@ ApiFuture<ResumableWrite> startResumableWrite(
             req);
   }
 
-  /**
-   * When using the retry features of the Gapic client, we are only allowed to provide a {@link
-   * Set}{@code <}{@link StatusCode.Code}{@code >}. Given {@link StatusCode.Code} is an enum, we can
-   * resolve the set of values from a given {@link ResultRetryAlgorithm} by evaluating each one as
-   * an {@link ApiException}.
-   */
-  static Set<StatusCode.Code> resultRetryAlgorithmToCodes(ResultRetryAlgorithm<?> alg) {
-    return CODE_API_EXCEPTIONS.stream()
-        .filter(e -> alg.shouldRetry(e, null))
-        .map(e -> e.apiExceptionCause.getStatusCode().getCode())
-        .collect(Collectors.toSet());
-  }
-
-  private static GrpcStatusCode statusCodeFor(StatusCode.Code code) {
-    switch (code) {
-      case OK:
-        return GrpcStatusCode.of(Code.OK);
-      case CANCELLED:
-        return GrpcStatusCode.of(Code.CANCELLED);
-      case UNKNOWN:
-        return GrpcStatusCode.of(Code.UNKNOWN);
-      case INVALID_ARGUMENT:
-        return GrpcStatusCode.of(Code.INVALID_ARGUMENT);
-      case DEADLINE_EXCEEDED:
-        return GrpcStatusCode.of(Code.DEADLINE_EXCEEDED);
-      case NOT_FOUND:
-        return GrpcStatusCode.of(Code.NOT_FOUND);
-      case ALREADY_EXISTS:
-        return GrpcStatusCode.of(Code.ALREADY_EXISTS);
-      case PERMISSION_DENIED:
-        return GrpcStatusCode.of(Code.PERMISSION_DENIED);
-      case RESOURCE_EXHAUSTED:
-        return GrpcStatusCode.of(Code.RESOURCE_EXHAUSTED);
-      case FAILED_PRECONDITION:
-        return GrpcStatusCode.of(Code.FAILED_PRECONDITION);
-      case ABORTED:
-        return GrpcStatusCode.of(Code.ABORTED);
-      case OUT_OF_RANGE:
-        return GrpcStatusCode.of(Code.OUT_OF_RANGE);
-      case UNIMPLEMENTED:
-        return GrpcStatusCode.of(Code.UNIMPLEMENTED);
-      case INTERNAL:
-        return GrpcStatusCode.of(Code.INTERNAL);
-      case UNAVAILABLE:
-        return GrpcStatusCode.of(Code.UNAVAILABLE);
-      case DATA_LOSS:
-        return GrpcStatusCode.of(Code.DATA_LOSS);
-      case UNAUTHENTICATED:
-        return GrpcStatusCode.of(Code.UNAUTHENTICATED);
-      default:
-        throw new IllegalStateException("Unrecognized status code: " + code);
-    }
-  }
-
-  private static ImmutableList<String> updateFields() {
```

Comment:
Did we move away from using this list?

Reply:
Yeah, in https://github.com/googleapis/java-storage/pull/1637 we added individual field level modification tracking so we don't guess.